package.json

```diff
@@ -24,6 +24,7 @@
   "dependencies": {},
   "peerDependencies": {
     "eslint": "^5.11.1",
+    "eslint-config-airbnb": "^18.0.1",
```

Comment:
This package is not required for base installation. Like for sdk or VoIP. With this change it is gonna warning all the time in these packages and "force" you to install something what is not required.